src/app/[locale]/vulnerabilities/detail/[id]/components/VulnerabilityDetailsTab.tsx

```diff
@@ -0,0 +1,152 @@
+// Copyright (C) Siemens AG, 2023. Part of the SW360 Frontend Project.
+
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+
+// SPDX-License-Identifier: EPL-2.0
+// License-Filename: LICENSE
+
+'use client'
+
+import { Col, Row, ListGroup, Tab } from 'react-bootstrap'
+import React, { useState, useEffect } from 'react'
+import PageSpinner from '@/components/Spinner/Spinner'
+
+import { useTranslations } from 'next-intl'
+import { COMMON_NAMESPACE } from '@/object-types/Constants'
+
+import ApiUtils from '@/utils/api/api.util'
+import { Session } from '@/object-types/Session'
+import HttpStatus from '@/object-types/enums/HttpStatus'
+import { signOut } from 'next-auth/react'
+
+import { VulnerabilitySummaryDataType } from '@/object-types/VulnerabilitySummaryDataType'
+import Summary from './Summary'
+
+import { notFound } from 'next/navigation'
+
+export default function VulnerabilityDetailsTab({
+    session,
+    vulnerabilityId,
+}: {
+    session: Session
+    vulnerabilityId: string
+}) {
+    const t = useTranslations(COMMON_NAMESPACE)
+    const [summaryData, setSummaryData] = useState<VulnerabilitySummaryDataType | undefined>(undefined)
+
+    useEffect(() => {
+        const controller = new AbortController()
+        const signal = controller.signal
+
+        ;(async () => {
+            try {
+                const response = await ApiUtils.GET(
+                    `vulnerabilities/${vulnerabilityId}`,
+                    session.user.access_token,
+                    signal
+                )
+                if (response.status === HttpStatus.UNAUTHORIZED) {
+                    return signOut()
+                } else if (response.status !== HttpStatus.OK) {
+                    return notFound()
+                }
+
+                const data = await response.json()
+
+                setSummaryData({
```

Comment:
You don't need to do this. You just need to use setSummaryData(data) and [conditional rendering](https://react.dev/learn/conditional-rendering) to render data in jsx.

Reply:
Done